### DISCUSSION TITLE
Animating elements on page load with SvelteKit

### DISCUSSION THREAD

**Discussion by matiadev** [sveltejs/kit#4804] [Q&A]
Using `onMount` prevents server-side rendering so I was wondering if using CSS animations is the only way to animate elements when a page loads?

```html
<script>
  import { onMount } from 'svelte'
  import { fade } from 'svelte/transition'

  let animate = false

  onMount(() => {
    animate = true
  })
</script>

{#if animate}
  <div transition:fade>
    <!-- ... -->
  </div>
{/if}
```

**Comment by geoffrich** [comment]
Related: https://github.com/sveltejs/kit/issues/2759

But yeah, not sure what this looks like.

**Reply by matiadev** [reply]
I would love that!
